rep-2000.rst

```diff
@@ -0,0 +1,121 @@
+REP: 2000
+Title: ROS 2.0 Target Platforms
+Author: Mikael Arguedas <mikael@openrobotics.org>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 10-Apr-2018
+
+
+Abstract
+========
+
+This REP defines target platforms for each ROS 2.0 Distribution Release.
+We define platforms to include both operating system releases (e.g. Ubuntu
+Xenial (16.04 LTS)) as well as major language releases (e.g. Python 3.5). The
+target platforms represent the set on which all core stacks are
+expected to work. Exceptions can be made for stacks that are
+intentionally platform-specific.
+
+Platforms by Distribution
+=========================
+
+If planned support for a target platform is changed, notice will be
+sent to ros-users to enable discussion of this issue.
+
+Ardent Apalone (December 2017 - December 2018)
+----------------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- MacOS El Capitan (10.11)
+- MacOS Sierra (10.12)
+- MacOS High Sierra (10.13)
```

Comment:
Do we really want to list all three? E.g. we don't even have a high sierra build node atm. I would keep the list rather small - that doesn't mean it doesn't work on not listed platforms - just that we don't actively test for it.

Reply:
Yeah I wasn't sure about that. At the time I wrote it we were testing the 3. I guess it;s a question of Where it's "expected to work". I didnt want to have separate columns for them but thought it could be useful to keep them here. I'm fine removing them though.

How do you suggest we let people know on which MacOS versions they should / should not try to install our binaries?

Reply:
If we test on `Sierra` we should only list that one in my opinion. Users can try it on any platforms they like.